Helper/Data.php

```diff
@@ -825,7 +825,6 @@ public function getOneClickPaymentMethods($customerId, $storeId, $grandTotal, $r
 
         $baCollection = $this->_billingAgreementCollectionFactory->create();
         $baCollection->addFieldToFilter('customer_id', $customerId);
-        $baCollection->addFieldToFilter('store_id', $storeId);
```

Comment:
why is this removed ( https://github.com/Adyen/adyen-magento2/blob/develop/Helper/Data.php#L808 )?